typings/index.d.ts

```diff
@@ -248,11 +248,14 @@ declare module 'discord.js' {
   }
 
   export class ClientApplication extends Application {
-    public botPublic: boolean | null;
-    public botRequireCodeGrant: boolean | null;
-    public cover: string | null;
-    public owner: User | Team | null;
+    public botPublic?: boolean;
+    public botRequireCodeGrant?: boolean;
+    public cover?: string;
+    public flags: number;
+    public owner?: User | Team;
+    public readonly partial: boolean;
     public rpcOrigins: string[];
+    public fetch(): Promise<ClientApplication>;
```

Comment:
Technically, we know that fetch call will return all props.. do we wanna `Required<ClientApplication>`?